src/pages/UserHome.jsx

```diff
@@ -109,35 +107,40 @@ export default function UserHome() {
             Add Scholarship
           </Button>
         </Grid>
+        {scholarships.length === 0 && (
+          <Grid container className={classes.bannerRoot}>
+            <Grid item xs={3}>
+              <InboxIcon style={{ fontsize: 600 }} />
+            </Grid>
+            <Grid item>
+              <Typography variant="h5" gutterButtom>
+                No Scholarships Added Yet{' '}
+              </Typography>
+              <Typography
+                variant="contained"
+                color="primary"
+                component={Link}
+                to="/scholarships/new">
+                Add Scholarship
+              </Typography>
```

Comment:
The Typography/MuiLink structure you had before was fine. Why the change?

Reply:
I left it like that because it didnt seem you were being strict about it and I thought it was not really necessary to change, but I addressed all of your other comments.

Reply:
Oh, well actually the _original_ code used the Typography/MuiLink structure. That's why I was asking why it was changed. It can be left the way you changed it as the styling for the page needs to be completed later anyways.